tasks/model-comparison/task.yaml

```diff
@@ -0,0 +1,22 @@
+id: adf92e59-35a6-47b9-b526-0d6615d74b4e
+name: Model Comparison
+description:
+  "Task that takes a list of Coretex models as input and compares them by their accuracy.\n
```

Comment:
First line of description should be on the same line as the field definition.
There is no need to manually enter a "\n" character. Yaml format supports them by default

Reply:
Auto save in VSC automatically add's those lines, but will try to edit it in simple text editor instead.